internal/server/resource/synonyms.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Should synonyms be bidirectional? so we also has "woman" to "female". 

Or should each synonyms be a set and we built the mapping from it?

Reply:
good idea, updated